packages/flutter/test/material/text_field_test.dart

```diff
@@ -363,6 +362,8 @@ void main() {
     expect(textField.cursorRadius, const Radius.circular(3.0));
   });
 
+  // TODO(hansmuller): restore these tests after the fix for #24876 has landed.
```

Comment:
Not sure what this comment means. Isn't this PR the fix for the fix for #24876?

Reply:
Yes. I was going to land the fix and then update the golden images and then update this test.